smbackend_turku/tests/test_utils.py

```diff
@@ -1,8 +1,55 @@
-from smbackend_turku.importers.utils import convert_code_to_int
+import os
+import json
+
+from django.conf import settings
+from django.contrib.gis.geos import Point
+
+from smbackend_turku.importers.utils import convert_code_to_int, get_weekday_str
 
 
 def test_convert_code_to_int():
     code_1 = '1_13'
     code_2 = '11_3'
     assert convert_code_to_int(code_1) == convert_code_to_int(code_1)
     assert convert_code_to_int(code_1) != convert_code_to_int(code_2)
+
```

Comment:
This file is not actually utils for tests but for testing utils, so you could create new file `utils.py ` and move these functions there.

Reply:
Also, `get_test_resource `, `format_time ` and `get_opening_hours ` are not used in code created in this commit, so commit them when they are used.